src/layouts/Base.astro

```diff
@@ -1,115 +1,116 @@
-          or click the GitHub logo to see its source code!
-        </p>
-      </div>
-    </PageTitle>
+	<body>
+		<PageTitle>
+			<Fragment slot="title">
+				<span>astro.new</span>{
+					pathname !== "" && (
+						<>
+							<span class="text-secondary-300 px-[0.1em] font-normal">/</span>
+							<span>{pathname.substring(1)}</span>
+						</>
+					)
+				}
+			</Fragment>
+			<div class="max-w-prose space-y-4">
+				<p><strong>Welcome to Astro’s official example projects!</strong></p>
+				<p>
+					Each of these templates is meant to show a common pattern or use case:
+					different kinds of websites, various integrations included
+					automatically, or how to work with common tools.
+				</p>
+				<p>
+					Open these in your choice of online code editor right in your browser,
+					or click the GitHub logo to see its source code!
+				</p>
+			</div>
+		</PageTitle>
 
-    <main
-      class="mx-auto flex max-w-screen-xl flex-col px-4 pt-8 sm:px-6 sm:pt-12 lg:px-8 lg:pt-16"
-    >
-      <ul class="templates mb-24 flex flex-col gap-24">
-        {
-          groups.map(([category, examples]) => {
-            return (
-              <li class="flex flex-col gap-8">
-                <h2
-                  class:list={[
-                    "text-2xl",
-                    { "sr-only": category === TOP_SECTION },
-                  ]}
-                >
-                  {category}
-                </h2>
-                <ul class="grid gap-4 sm:grid-cols-2 lg:grid-cols-3 lg:gap-6">
-                  {examples.map(
-                    (
-                      example: astroHTML.JSX.IntrinsicAttributes & CardProps
-                    ) => (
-                      <li>
-                        <Card {...example} />
-                      </li>
-                    )
-                  )}
-                  {category === TOP_SECTION && <Outlinks />}
-                </ul>
-              </li>
-            );
-          })
-        }
-      </ul>
-    </main>
-  </body>
+		<main
+			class="mx-auto flex max-w-screen-xl flex-col px-4 pt-8 sm:px-6 sm:pt-12 lg:px-8 lg:pt-16"
+		>
+			<ul class="templates mb-24 flex flex-col gap-24">
+				{
+					groups.map(([category, examples]) => {
+						return (
+							<li class="flex flex-col gap-8">
+								<h2
+									class:list={[
+										"text-2xl",
+										{ "sr-only": category === TOP_SECTION },
+									]}
+								>
+									{category}
+								</h2>
+								<ul class="grid gap-4 sm:grid-cols-2 lg:grid-cols-3 lg:gap-6">
+									{examples.map(
+										(
+											example: astroHTML.JSX.IntrinsicAttributes & CardProps,
+										) => (
+											<li>
+												<Card {...example} />
+											</li>
+										),
+									)}
+									{category === TOP_SECTION && <Outlinks />}
+								</ul>
+							</li>
+						)
+					})
+				}
+			</ul>
+		</main>
+	</body>
 </html>
 
 <style>
-  body {
-    @apply bg-neutral-50 text-neutral-900;
```

Comment:
nit: lets leave this as dark text on a light background for now and update styles in a separate PR